docs/tutorials/pre_executed/zubercal-ps1-snad.ipynb

```diff
@@ -0,0 +1,2344 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        .rename(columns={"R.A.": "ra", "Dec.": "dec"})
</pre><p>Is there a problem with doing <code>from_dataframe(..., ra_column="R.A", dec_column="Dec.") </code> If yes. I'm guessing it's those periods in the name... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I think it was actually fixed in <code>nested-pandas</code>!</p>